src/core/typed_list.pmt

```diff
@@ -951,6 +951,45 @@ igraph_error_t FUNCTION(sort_ind)(
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup vector_list
+ * \function igraph_vector_list_remove_consecutive_duplicates
+ * \brief Removes consecutive duplicates from a vector list.
+ *
+ * Takes an unsorted list \p v as input and computes an array of
+ * indices \p inds such that v[ inds[i] ], with i increasing from 0, is
```

Comment:
It looks like this is not true any more. The current implementation seems to simply compare consecutive items and removes duplicates in-place. The documentation needs to be updated. We can mention that this actually removes all duplicates if the list is pre-sorted first.

Reply:
Thanks, copy-paste mistake!

Reply:
Allowing `NULL` is actually not easily possible because the same framework is used for lists of `igraph_t` as well. That case is different enough that I wouldn't want to allow a default interpretation just yet.